### PR TITLE
add possibility to reserve tickets

### PR DESCRIPTION
**Pull request by Marco-Santoni**
Solves #20

### CODE REVIEW

File: layouts/partials/tickets.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!-- FILEPATH: /c:/Users/MarcoSantoni/py4ai/layouts/partials/tickets.html -->
+<section class="bar background-white">
+    <div class="container">
+        <div class="heading text-center">
+            <h2>Tickets</h2>
+        </div>
+
+        <!-- place this in your head tag -->
+        <script src='https://js.tito.io/v2' async></script>
```
Comment (pietroppeter) [inline layouts/partials/tickets.html]: this should be moved in the customer_heders file

**Review by pietroppeter — CHANGES_REQUESTED** [review]
Hi, thanks! Just a small change request, the rest looks good! Can you share a screenshot on how it looks?

**Comment by Marco-Santoni** [comment]
This is how ti looks 
image

**Comment by pietroppeter** [comment]
Thanks!

**Pull request merged**